examples/orpc/src/index.ts

```
@@ -0,0 +1,170 @@
+  .use(evlog())
+
+type Role = 'guest' | 'admin' | 'superadmin'
+
+interface AuthedUser {
+  id: string
+  name: string
+  role: Role
+  apiKey: string
+}
+
+const authed = base.use(async ({ context, next }) => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)